sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -377,10 +391,10 @@ static const std::pair<int, int> getRangeOfAllowedComputeEngines = [] {
 // available copy engines can be used.
 static const std::pair<int, int> getRangeOfAllowedCopyEngines = [] {
   const char *EnvVar = std::getenv("SYCL_PI_LEVEL_ZERO_USE_COPY_ENGINE");
-  // If the environment variable is not set, all available copy engines can be
+  // If the environment variable is not set, only index 0 copy engine will be
   // used.
   if (!EnvVar)
-    return std::pair<int, int>(0, INT_MAX);
+    return std::pair<int, int>(0, 0);
```

Comment:
Let's only do that for immediate submissions in this PR (to avoid regressions unrelated to these changes)

Reply:
Hmm, I thought I did that using a check if immediate is being used.